deps/build.jl

```diff
@@ -58,8 +63,7 @@ provides(AptGet,
         "libpango1.0-0" => [pango,pangocairo],
         "libglib2.0-0" => gobject,
         "libpng12-0" => libpng,
-        "libpixman-1-0" => pixman,
-        "gettext" => gettext
```

Comment:
I also wanted to remove this some time ago, but it'ss actually needed by GLib. Though I'm not sure anybody has already tested building everything from source. See https://github.com/JuliaLang/Cairo.jl/issues/36.